### ISSUE TITLE
When new MFF computers come in, test software infrastructure installation procedure

### ISSUE THREAD

**Issue by BradSchoenrock** [NCAR/WVD-MCSupdate#60]
5 MFF computers are on their way. When we receive them, install on one computer and check if it is hot swap-able with cuddlefish (current MFF). Once that is done Robert can use cuddlefish for development so we can stop having LabView 2012 vs 2014 compatibility issues that have plagued our development. 

Then we can reassess the necessary pieces of our software infrastructure by adding the software stack to each computer, itterating as we go. Eventually we will wipe cuddlefish for operation of a WVD system.

**Comment by BradSchoenrock** [comment]
added bug tag because getting robert a windows machine with labview 2012 will resolve version conflicts bug which has been reoccurring.

**Comment by BradSchoenrock** [comment]
Scott Spuler - MMF computers should arrive 1-Dec

**Comment by BradSchoenrock** [comment]
MFF computers arrived today, Scott has contacted Ted (i think) to get windows on the machines, and we can work on getting our programs (labview, matlab, etc...) onto them when robert gets back next week.

**Comment by BradSchoenrock** [comment]
Robert has installs going for a couple new machines and we will work towards hot swapping one of them this week.

**Comment by stillwer** [comment]
All programs should be installed on eol-dial1 and eol-dial2. I just took all the screen shots needed to walk someone through the process. I plan to compile it into a document and give it to Brad so see if he can replicate the steps.

**Comment by BradSchoenrock** [comment]
link to document in google docs ... 

https://docs.google.com/document/d/1Tx8YbyNrgh-XaWgp-6rkG9MQMIqk4Xktu_VzuE5KCmY/edit

**Comment by BradSchoenrock** [comment]
I've claimed eol-dial4 for testing install procedure.

**Comment by BradSchoenrock** [comment]
i have sucessfully installed our needed software on eol-dial4 and updated documentation to reflect the installation procedure most notably for git code, malwarebytes, and python. 

This should close the issue for now, if we want to change the software distribution method in the future that will be opened in another issue.
